### PR TITLE
fix(nala): repo and key not matching upstream changes

### PR DESCRIPTION
**Pull request by volitank**

### CODE REVIEW

**Comment by volitank** [comment]
Resolves: https://github.com/wimpysworld/deb-get/issues/1098

**Comment by philclifford** [comment]
Thanks! That should fix our issue. 
BTW - in testing I ran across this problem with the installed nala : 
```
nala update
Traceback (most recent call last):
  File "/usr/bin/nala", line 5, in <module>
    from nala.__main__ import main
  File "/usr/lib/python3/dist-packages/nala/__main__.py", line 31, in <module>
    import nala.fetch as _fetch  # pylint: disable=unused-import
  File "/usr/lib/python3/dist-packages/nala/fetch.py", line 36, in <module>
    import typer
  File "/usr/lib/python3/dist-packages/typer/__init__.py", line 12, in <module>
    from click.termui import get_terminal_size as get_terminal_size
ImportError: cannot import name 'get_terminal_size' from 'click.termui' (/usr/lib/python3/dist-packages/click/termui.py)
```
for every invocation, including bash auto-complete ... (on up-to-date  Ubuntu Jammy)

File: 01-main/packages/nala [hunk 1]
```diff
@@ -1,8 +1,8 @@
 DEFVER=1
```
Comment (philclifford) [inline 01-main/packages/nala:1]:
```suggestion
DEFVER=2
```

**Review by philclifford — APPROVED** [review]

**Comment by philclifford** [comment]
> nala update
> Traceback (most recent call last):
> [...]
> ImportError: cannot import name 'get_terminal_size' from 'click.termui' (/usr/lib/python3/dist-packages/click/termui.py)


This can be fixed by downgrading the click package back to the ubuntu version with 

```
sudo apt install --reinstall python3-click=8.0.3-1
```

**Pull request merged**